decidim-core/app/assets/stylesheets/decidim/_decidim.scss

```diff
@@ -1,9 +1,14 @@
 @import "variables";
 @import "foundation";
-@import "utils/*";
+@import "utils/settings";
 
 @include foundation-everything($flex: false);
```

Comment:
A couple of things here:
1. I had to move & remove the wildcard to get the foundation styles properly.
2. I do not understand why:
    1. Foundation settings are inside `utils` folder. They're not an "utility".
    2. `foundation-everything($flex: false)` any reason to set flex to false?
    3. `utils, layouts, extras` folders are included throught wildcards. `Modules` includes a modules.scss file, instead. How come?

Reply:
I know the answer to ii. Latest foundation is "flex by default" but the previous version was not. So when I upgraded I had to specify `$flex: false` so that the layout wouldn't break. Of course we can migrate to `$flex: true` but I think that would need some work.

Reply:
@Crashillo most of these issues are probably just legacy and product of rushed decisions when bootstrapping the project. Feel free to refactor it the way you find more comfortable.